packages/react/src/components/InputField/index.js

```diff
@@ -1,13 +1,31 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import FormField from '../FormField';
 import Input from '../Input';
 
-const InputField = React.forwardRef((props, ref) => (
-  <FormField {...props} ref={ref}>
-    <Input {...props} />
-  </FormField>
-));
+const InputField = React.forwardRef(
+  ({ label, helper, inputRef, ...props }, ref) => (
+    <FormField label={label} helper={helper} {...props} ref={ref}>
+      <Input ref={inputRef} {...props} />
```

Comment:
If we sending props to both `FormField` and `Input` will they not get the same required `id`?

Reply:
That's the point 🙂 
The `FormField` needs this id for the `Label`
https://github.com/heetch/flamingo/blob/master/packages/react/src/components/FormField/index.js#L11

Whereas the `Input` will use it as a regular id `id={id}`

Reply:
Cool got you, I see `FormField` is using it for the label, thanks